packages/terra-functional-testing/src/terra-cli/wdio/test-runner.js

```diff
@@ -73,13 +79,19 @@ class TestRunner {
    * @param {string} options.formFactors - A list of form factors for the test run.
    * @param {string} options.locales - A list of language locales for the test run.
    * @param {string} options.themes - A list of themes for the test run.
+   * @param {string} options.hostname - Automation driver host address.
+   * @param {number} options.port - Automation driver port.
+   * @param {string} options.baseUrl - The base url.
+   * @param {array} options.suite - Overrides specs and runs only the defined suites.
+   * @param {array} options.spec - A list of spec file paths.
    */
   static async start(options) {
     const {
       config,
       formFactors = [],
       locales,
       themes,
+      ...launcherOptions // hostname, port, baseUrl, suite, and spec
```

Comment:
This is a sneaky way to pass the wdio-cli options without defining keys. We don't want to deconstruct the keys otherwise we have to filter them back out. Deconstructing leads to undefined values. 
```
// We don't want to pass undefined keys
{ host: undefined } 
```